cram/cram_io.c

```diff
@@ -5453,14 +5451,6 @@ int cram_seek(cram_fd *fd, off_t offset, int whence) {
     if (!(whence == SEEK_CUR && offset >= 0))
```

Comment:
This should always return -1 now, as the conditional only makes sense with the bit of following code that was removed.